docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+
+Follow the steps above to create additional contact points. Different contact points require different configuration information. For example, for Slack, PMM requires the recipient information, the API token and the webhook URL, which you can get from your Slack administrator.
 
 ### Notification policies
-Notification policies determine how Grafana alerts are routed to contact points by setting where, when, and how to send notifications. 
+
+Notification policies determine how Grafana alerts are routed to contact points by defining where, when, and how to send notifications.
 
 For example, you might specify a limit for the number of times a notification is sent during a certain period. This helps ensure that you don't spam your Slack channel with too many notifications about the same issue.
 
+#### PMM Root Notification policy
+
+Percona Alerting comes pre-configured with a Notification Root Policy, which is the default notification policy that PMM applies to all the alerts that don’t have a custom notification policy assigned to them. This is the **grafana-default-email** policy, which uses Email as the default contact point.
+
+#### How matching works
+
+Policies can have one or more child policies. An alert matches if the alert’s labels match all the **Matching Labels** specified on the policy.
+
+Alerts are first verified against the main policy, then against any matching child policies if the **Continue matching subsequent sibling nodes** is enabled.
+
+Alerts that don't match any specific policies are handled by the root policy. The root policy also handles any alert rules for which the assigned custom notification policy has been deleted, to ensure notifications for existing alerts continue to be delivered.
+
+#### Edit the root notification policy
+
+1. Go to {{icon.bell}} **Alerting > Notification policies** tab.
+2. Click **Edit** on the top right of the root policy box.
+3. Choose whether to keep the default Email contact point, select a new available contact point or create a new one.
+4. In the **Group by** field, specify how alert rules should be processed into notifications. If multiple alerts are matched for this policy, they will be grouped based on the labels you specify, and a notification will be sent per group.
+5. Expand the **Timing options** section and specify how notification wait times should be processed. These are short pauses the system can take to efficiently process multiple sets of alerts for notifications:
+   - **Group wait**: The default is to wait 30 seconds to buffer alerts of the same group before sending a notification initially.
+   - **Group interval**: The default is to wait five minutes before sending a batch of new alerts after the first notification was sent.
+   - **Repeat interval**: The default is to wait four hours before resending an alert.
+6. Click **Save** to save your changes.
+
```

Comment:
I would add a section on "specific routing" - specifically the ability to search by contact point - this is a very convenient way to show which contact points are associated with a notification policy